taskfiles/k8s.yml

```diff
@@ -203,17 +203,24 @@ tasks:
           fi
           echo $repo
     cmds:
+      - defer: 'docker logout'
       - cmd: docker login --username {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}
         silent: true
       - 'echo "~~~ :docker: Build & Push operator container"'
       - docker buildx build
```

Comment:
Could we reuse `build-operator-images` here? You'd need to make it take an argument for that tags.

Reply:
Yes, I can. It will be done in the next push.